apps/cli/src/commands/boot-challenger.ts

```diff
@@ -114,26 +115,78 @@ const onAssertionConfirmedCb = async (nodeNum: any, commandInstance: Vorpal.Comm
         }
     }
 
-    const assertionNode = await getAssertion(nodeNum);
-    commandInstance.log(`[${new Date().toISOString()}] Assertion data retrieved. Starting the submission process...`);
-    try {
-        await submitAssertionToReferee(
-            cachedSecretKey,
-            nodeNum,
-            assertionNode,
-            cachedSigner!.signer,
-        );
-        commandInstance.log(`[${new Date().toISOString()}] Submitted assertion: ${nodeNum}`);
-        lastAssertionTime = Date.now();
-    } catch (error) {
-        if (error && (error as Error).message && (error as Error).message.includes('execution reverted: "9"')) {
-            commandInstance.log(`[${new Date().toISOString()}] Could not submit challenge because it was already submitted`);
-            lastAssertionTime = Date.now();
-            return;
+    // Get Last Challenge Data
+    const challengeData = await getLatestChallenge();
+    const lastChallenge = challengeData[1];
+
+    const lastAssertionId = Number(lastChallenge.assertionId);
+    const time = Number(lastChallenge.assertionTimestamp);
+
+    // Calculate the minimum time to submit an assertion
+    const minimumTimeToSubmit = time + MINIMUM_TIME_BETWEEN_ASSERTIONS;
+
+    // Check if enough time has passed that we can submit an assertion
+    if(Date.now() > minimumTimeToSubmit) {
```

Comment:
Consider running a second instance. It would get the event, then wait for the instance delay and then the timestamp could be long enough for the 2nd instance to submit, not sure if thats what we want, we might have to confirm with Mark ?

Reply:
Will ask Mark at standup